companion/manifest.json

```
@@ -0,0 +1,31 @@
+{
+	"id": "phillips-hue",
+	"name": "philips-hue",
+	"shortname": "hue",
+	"description": "Philips Hue module for Companion",
+	"version": "0.2.0",
+	"license": "MIT",
+	"repository": "git+https://github.com/bitfocus/companion-module-phillips-hue.git",
+	"bugs": "https://github.com/bitfocus/companion-module-phillips-hue/issues",
+	"maintainers": [
+		{
+			"name": "Dominik Zipperle",
+			"email": "dominik.zipperle@googlemail.com"
+		}
+	],
+	"runtime": {
+		"type": "node18",
+		"api": "nodejs-ipc",
+		"apiVersion": "1.0.0",
```

```suggestion
		"apiVersion": "0.0.0",
```

This is why `configUpdated` is never called